data/json/achievements.json

```diff
@@ -484,15 +484,15 @@
     ]
   },
   {
-    "id": "achievement_mutation_threshold_cephalapod",
```

Comment:
I think you cannot change the id, as it would require migration.

Reply:
How difficult is it to migrate it?

Reply:
In my opinion it's better/easier not to. You'll probably have found that the item `balaclava` has it's id mispelled as `balclava` - it's been that way for a very long time because it's not very useful to change.

@jbytheway May have opinions on achievement stuff more, however.

Reply:
There's no support for migrating achievements as yet, so indeed I recommend not changing it.

Reply:
That's unfortunate. It's been reverted.

Reply:
But item IDs have a way to migrate them, don't they? Can we collect them in some issue and migrate in batch before some release?

Reply:
#46289